tfrog-motordriver/controlVelocity.c

```diff
@@ -109,7 +109,7 @@ void ISR_VelocityControl()
           motor[i].ref.vel_diff = 0;
         }
 
-        motor[i].error = motor[i].ref.vel_buf - motor[i].vel;
+        motor[i].error = motor[i].ref.vel - motor[i].vel;
```

Comment:
It seems better to leave this line as an old one.
Because if the interrupt occurs between these two lines https://github.com/T-frog/tf-2md3-firmware/pull/85/files#diff-bc1342d54328fc8b17402447f259b6dbR1529-R1530 , ```motor[i].ref.vel``` will not update but use new reference velocity.

Reply:
@seiga-k I think `ref.vel` can be updated as soon as possible.
Since `ref.vel_changed` flag is used to calculate acceleration of the reference for feed-forward dynamics compensation, it's not matter if `ref.vel_buf` has single step delay against `ref.vel`.
What do you think?

Reply:
Yes, it makes only a single step delay.
I thought the priority of the matching the ```ref.vel``` and ```ref.vel_diff``` is higher than the update delay.
This is only a balance of priority. I understood your opinion now, so I can agree with you.

Reply:
Another reason is that I would like to minimize possibility to miss updating the reference velocity, even if any other bugs are injected around `ref.vel_changed`.